tests/model_serving/model_server/authentication/conftest.py

```diff
@@ -135,6 +210,29 @@ def patched_remove_authentication_isvc(
         yield http_s3_caikit_serverless_inference_service
 
 
+@pytest.fixture()
+def patched_remove_raw_authentication_isvc(
+    admin_client: DynamicClient,
+    http_s3_caikit_raw_inference_service: InferenceService,
+) -> InferenceService:
+    with ResourceEditor(
+        patches={
+            http_s3_caikit_raw_inference_service: {
+                "metadata": {
+                    "labels": {"security.opendatahub.io/enable-auth": "false"},
```

Comment:
please save the label in `utilities.contants` ; create a new class there called
`Labels' and a sub-class for `Kserve`
and re-use in all places